frappe/desk/page/setup_wizard/setup_wizard.js

```diff
@@ -125,7 +120,7 @@ frappe.setup.SetupWizard = class SetupWizard extends frappe.ui.Slides {
 			return;
 		}
 		super.show_slide(id);
-		frappe.set_route(this.page_name, id + "");
+		frappe.set_route(this.page_name, cstr(id));
```

Comment:
Technically any one of the 3 changes I made fixes the issue. All 3 for good measure.